package.json

```diff
@@ -6,11 +6,11 @@
   "scripts": {
     "dev-api": "pnpm --filter api dev",
     "dev-web": "pnpm --filter web dev",
-    "build": "./scripts/build",
+    "build": "turbo build",
```

Comment:
I noticed that turbo caches things (yay for speed!). However, we only use this build command for deploying to npm and I think in that case we want to force it to build from scratch each time. Can we add the `--force` here?

(I already noticed that if the build was cached but I manually delete one of the built files, it doesn't get rebuilt because turbo thinks it hasn't changed. This makes me paranoid for npm publishing).

Reply:
turbo build gets used under the hood for the other commands, I would recommend that when publishing to npm you use `--force` but if I add `--force` here then every test, link, etc would force a build of everything every time

Reply:
Ahh ok.